generation/2d_ddpm/2d_ddpm_tutorial.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>The same as before here, perhaps need add generative repo link here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>